src/components/ValidatorForm.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import { ResourceForm } from './ResourceForm';
+
+import { SelectOption } from '../models/SelectOption';
+import { ProfileForm } from './ProfileForm';
+
+export interface ValidatorProps {
+  readonly basePath: string;
+  readonly profiles: Map<string, string[]>;
```

Comment:
I think this should probably be a record

```suggestion
  readonly profiles: Record<string, string[]>;
```